plugins/content-length.rst

```diff
@@ -1,4 +1,38 @@
 Content-Length Plugin
 =====================
 
-TODO
+The Content-Length Plugin allows you to make your HTTP Request compliant with most of the servers.
+It's main purpose is to set the correct `Content-Length` Header value based on your stream size.
```

Comment:
The `ContentLenghtPlugin` sets the correct `Content-Length` header value based on the size of the body stream of the request. This helps HTTP servers to handle the request.